practice/spec/regex/regex_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe 'Regex pattern' do
+  before(:each) do
+    @book_name = 'Story.txt'
+    @file_name = 'alphanumeric.txt'
```

Comment:
As I mentioned in a comment above, we want have the full path to the file to put it as argument for count_occurences method.